Example/Example/Modules/Objective-C/WrapFaroController.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
```

Comment:
Die hadden we daarvoor ook. Faro kan gebruikt worden in Objective-C. Dit is gewoon een voorbeeld.